qiita_ware/ebi.py

```diff
@@ -543,46 +476,31 @@ def generate_run_xml(self):
             data_block = ET.SubElement(run, 'DATA_BLOCK')
             files = ET.SubElement(data_block, 'FILES')
             ET.SubElement(files, 'FILE', {
-                'filename': join(self.ebi_dir, basename(file_path)),
+                'filename': file_path,
                 'filetype': file_type,
                 'quality_scoring_system': 'phred',
                 'checksum_method': 'MD5',
                 'checksum': md5}
             )
 
-        return run_set
+        return ET.tostring(run_set)
 
-    def generate_submission_xml(self, action):
+    def generate_submission_xml(self, date_to_hold=date.today()):
         """Generates the submission XML file
 
         Parameters
         ----------
-        action : {'ADD', 'VALIDATE', 'UPDATE'}
-            What action to take when communicating with EBI
+        date_to_hold : date, optional
+            Date to be added to submission. Useful for testing.
```

Comment:
This is more than the date added to the submission, this determines the date until which the submission will be held private at EBI's repository i.e. 365 days after this date. Would you mind adding that information in the other generate_*_xml methods?